integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -43,28 +43,25 @@ object HiveExample {
 
     import org.apache.spark.sql.CarbonSession._
 
-    val carbonSession = SparkSession
-      .builder()
-      .master("local")
-      .appName("HiveExample")
-      .config("carbonSession.sql.warehouse.dir", warehouse).enableHiveSupport()
-      .getOrCreateCarbonSession(
-        store, metaStore_Db)
+    val carbonSession = SparkSession.builder()
+      .master("local").appName("HiveExample")
```

Comment:
Please keep it，no need to change

Reply:
Yes,I replace this with ExampleUtil to create carbon session.

Reply:
ok